Main.elm

```diff
@@ -1,7 +1,393 @@
+
+        FetchRecordsResponse (Ok recordList) ->
+            { model
+                | records = Just recordList
+                , error = Nothing
+            }
+                ! []
+
+        FetchRecordsResponse (Err error) ->
+            { model | error = Just <| toString error } ! []
+
+        Logout ->
+            { model
+                | error = Nothing
+                , records = Nothing
+                , bearer = Nothing
+            }
+                ! [ saveData { key = "bearer", value = Encode.null } ]
+
+        AcceptStep recordId ->
+            model ! [ answerStep model recordId "SUCCEED" ]
+
+        RejectStep recordId ->
+            model ! [ answerStep model recordId "FAIL" ]
+
+        AnswerResponse (Err error) ->
+            { model | error = Just <| toString error } ! []
+
+        AnswerResponse (Ok _) ->
+            model ! [ fetchRecordList model ]
+
+        Tick _ ->
+            model ! [ fetchRecordList model ]
+
+
+
+-- Kinto related
+
+
+answerUrl : String -> String -> String -> String -> Url
+answerUrl baseUrl bucketName collectionName recordId =
+    let
+        url =
+            if String.endsWith "/" baseUrl then
+                String.dropRight 1 baseUrl
+            else
+                baseUrl
+
+        joinUrl =
+            String.join "/"
+    in
+        joinUrl
+            [ url
+            , "buckets"
+            , bucketName
+            , "collections"
+            , collectionName
+            , "records"
+            , recordId
+            , "stepfunction"
+            ]
+
+
+answerResource : String -> String -> String -> Encode.Value -> Kinto.Client -> HttpBuilder.RequestBuilder ()
+answerResource bucketName collectionName recordId body client =
+    answerUrl client.baseUrl bucketName collectionName recordId
+        |> HttpBuilder.post
+        |> HttpBuilder.withHeaders client.headers
+        |> HttpBuilder.withJsonBody body
+
+
+encodeAnswer : String -> Encode.Value
+encodeAnswer answer =
+    Encode.object [ ( "answer", Encode.string answer ) ]
+
+
+answerStep : Model -> RecordId -> String -> Cmd Msg
+answerStep model recordId answer =
+    case model.bearer of
+        Nothing ->
+            Cmd.none
+
+        Just bearer ->
+            let
+                client =
+                    Kinto.client model.kintoServer (Kinto.Custom "Portier" bearer)
+            in
+                client
+                    |> answerResource model.kintoBucket model.kintoCollection recordId (encodeAnswer answer)
+                    |> Kinto.send AnswerResponse
+
+
+fetchRecordList : Model -> Cmd Msg
+fetchRecordList model =
+    case model.bearer of
+        Nothing ->
+            Cmd.none
+
+        Just bearer ->
+            let
+                client =
+                    Kinto.client model.kintoServer (Kinto.Custom "Portier" bearer)
+            in
+                client
+                    |> Kinto.getList (recordResource model.kintoBucket model.kintoCollection)
```

Comment:
`recordResource` could be set in the `let` above, removing the "need" for the `recordResource` below